Sources/AppCoinsSDK/Domain/Entities/GuestWallet.swift

```diff
@@ -22,11 +22,11 @@ internal class GuestWallet: Wallet, Codable {
         self.signature = signature
     }
 
-    func getBalance(currency: Coin, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
+    func getBalance(currency: String, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void) {
```

Comment:
This should follow the same logic of the comments on the `getBalance()` method on the `ClientWallet.swift` file.

Reply:
Done